source/common/quic/http_datagram_handler.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "envoy/http/codec.h"
+
+#include "source/common/common/assert.h"
+#include "source/common/common/logger.h"
+#include "source/common/http/header_map_impl.h"
+
+#include "absl/strings/string_view.h"
+#include "quiche/common/simple_buffer_allocator.h"
+#include "quiche/quic/core/http/quic_spdy_stream.h"
+
+namespace Envoy {
+namespace Quic {
+
+// Enables HTTP Datagrams and the Capsule Protocol support based on RFC 9297. This class is used
+// both on the receive side (HTTP/3 codec -> Envoy) and the send side (Envoy -> HTTP/3 codec) of
```

Comment:
FYI: "Receive side" and "Send side" are perfectly reasonable terms to use here, though Decode and Encode are the specific Envoy terms for these operations.

Reply:
Done.